build.sbt

```diff
@@ -54,21 +44,15 @@ stage := {
   }
 }
 
-lazy val root = project
-  .in(file("."))
+lazy val root = tlCrossRootProject
```

Comment:
What does it mean `tlCrossRootProject`? Will they all be cross compiled to js/jvm? If so it doesn't seem correct as e.g. explore is js only

Reply:
Actually, some modules in explore are JVM and JS.
https://github.com/gemini-hlsw/explore/blob/a366ec0a2be4df7a844c45ffb8da1313f258c489/build.sbt#L63

All that the `tlCrossRootProject` does is take a list of modules, and sort them into a JVM CI job and a JS CI job.
https://typelevel.org/sbt-typelevel/faq.html#how-do-i-split-my-ci-matrix-into-separate-jobs-for-jvm-js-etc

Reply:
Yes, I meant the sub project `explore` is js only, basically anything that takes `scalajs-react` is js only

Reply:
`tlCrossRootProject` is syntax sugar for:

```scala
lazy val root = project.in(file("."))
  .enablePlugins(NoPublishPlugin)
  .aggregate(model.jvm, model.js, modelTests.jvm, modelTests.js, graphql, common, explore)

lazy val rootJS = project.in(file("."))
  .aggregate(model.js, modelTests.js, graphql, common, explore)

lazy val rootJVM = project.in(file("."))
  .enablePlugins(NoPublishPlugin)
  .aggregate(model.jvm, modelTests.jvm)

ThisBuild / githubWorkflowBuildMatrixAdditions += "project" -> List("rootJS", "rooyJVM")
ThisBuild / githubWorkflowBuild := Seq(WorkflowStep.Sbt(List("project ${{ matrix.project }}", "test"))
// etc. etc. etc.
```